Calibration/HcalCalibAlgos/src/Analyzer_minbias.cc

```diff
@@ -11,736 +11,434 @@
-   myTree->Branch("mom4_Noise",  &mom4_Noise, "mom4_Noise/F");
-   
-   myTree->Branch("mom0_Diff",  &mom0_Diff, "mom0_Diff/F");
-   myTree->Branch("mom1_Diff",  &mom1_Diff, "mom1_Diff/F");
-   myTree->Branch("mom2_Diff",  &mom2_Diff, "mom2_Diff/F");
-
-   myTree->Branch("occup",  &occup, "occup/F");
-   
-   std::cout<<" Before ordering Histos "<<std::endl;
   
-   char str0[15];
-   char str1[15];
-
-   char str10[15];
-   char str11[15];
-
-   int k=0;
-   nevent = 0;
-// Size of collections
-
-   hHBHEsize_vs_run = new TH2F("hHBHEsize_vs_run","hHBHEsize_vs_run",500,111500.,112000.,6101,-100.5,6000.5);
-   hHFsize_vs_run = new TH2F("hHFsize_vs_run","hHFsize_vs_run",500,111500.,112000.,6101,-100.5,6000.5);
+  void Analyzer_minbias::beginJob() {
+    hOutputFile   = new TFile( fOutputFileName.c_str(), "RECREATE" ) ;
+
+    myTree = new TTree("RecJet","RecJet Tree");
+    myTree->Branch("mydet",  &mydet, "mydet/I");
+    myTree->Branch("mysubd",  &mysubd, "mysubd/I");
+    myTree->Branch("cells", &cells, "cells");
+    myTree->Branch("depth",  &depth, "depth/I");
+    myTree->Branch("ieta",  &ieta, "ieta/I");
+    myTree->Branch("iphi",  &iphi, "iphi/I");
+    myTree->Branch("eta",  &eta, "eta/F");
+    myTree->Branch("phi",  &phi, "phi/F");
+    myTree->Branch("mom0_MB",  &mom0_MB, "mom0_MB/F");
+    myTree->Branch("mom1_MB",  &mom1_MB, "mom1_MB/F");
+    myTree->Branch("mom2_MB",  &mom2_MB, "mom2_MB/F");
+    myTree->Branch("mom4_MB",  &mom4_MB, "mom4_MB/F");
+    myTree->Branch("mom0_Noise",  &mom0_Noise, "mom0_Noise/F");
+    myTree->Branch("mom1_Noise",  &mom1_Noise, "mom1_Noise/F");
+    myTree->Branch("mom2_Noise",  &mom2_Noise, "mom2_Noise/F");
+    myTree->Branch("mom4_Noise",  &mom4_Noise, "mom4_Noise/F");
+    myTree->Branch("mom0_Diff",  &mom0_Diff, "mom0_Diff/F");
+    myTree->Branch("mom1_Diff",  &mom1_Diff, "mom1_Diff/F");
+    myTree->Branch("mom2_Diff",  &mom2_Diff, "mom2_Diff/F");
+    myTree->Branch("occup",  &occup, "occup/F");
+    myTree->Branch("trigbit", &trigbit, "trigbit/I");
+    myTree->Branch("rnnumber", &rnnumber, "rnnumber/D");
+
+    myMap.clear();
+    return ;
+  }
+  
+  //  EndJob
+  //
+  void Analyzer_minbias::endJob() {
    
-   for(int i=1;i<73;i++){
-    for(int j=1;j<43;j++){
-
-       meannoise_pl[i][j] = 0.;
-       meannoise_min[i][j] = 0.;
-
-//     for(int l=1;l<5;l++){
-        k = i*1000+j;
-        sprintf(str0,"mpl%d",k);
-        sprintf(str1,"mmin%d",k);
-
-        sprintf(str10,"vpl%d",k);
-        sprintf(str11,"vmin%d",k);
-//      cout<<" "<<i<<" "<<j<<endl;
-    if( j < 30 )
-    {
-// first order moment
-    hCalo1[i][j] = new TH1F(str0, "h0", 320, -10., 10.);
-    hCalo2[i][j] = new TH1F(str1, "h1", 320, -10., 10.);
-
-// second order moment
-    hCalo1mom2[i][j] = new TH1F(str10, "h10", 320, 0., 20.);
-    hCalo2mom2[i][j] = new TH1F(str11, "h11", 320, 0., 20.);
+    int ii=0;
+    // std::cout << " Hello me here" << std::endl;
```

Comment:
Hi

I am surprized - because these lines of code are from the earlier version. The new version of Analyzer_minbias.cc does not have double indexed arrays of TH1F*. I removed the commented std::cout statements from the code and made a new push with the earlier pull request. Regards

Sunanda
________________________________
From: Alessandro Degano [notifications@github.com]
Sent: 26 January 2015 10:48
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] Run2-HcalPhiSymmetry-nzs01 (#7337)


In Calibration/HcalCalibAlgos/src/Analyzer_minbias.cc<https://github.com/cms-sw/cmssw/pull/7337#discussion_r23519615>:

> -        sprintf(str1,"mmin%d",k);
> -
> -        sprintf(str10,"vpl%d",k);
> -        sprintf(str11,"vmin%d",k);
> -//      cout<<" "<<i<<" "<<j<<endl;
> -    if( j < 30 )
> -    {
> -// first order moment
> -    hCalo1[i][j] = new TH1F(str0, "h0", 320, -10., 10.);
> -    hCalo2[i][j] = new TH1F(str1, "h1", 320, -10., 10.);
> -
> -// second order moment
> -    hCalo1mom2[i][j] = new TH1F(str10, "h10", 320, 0., 20.);
> -    hCalo2mom2[i][j] = new TH1F(str11, "h11", 320, 0., 20.);
> +    int ii=0;
> +    // std::cout << " Hello me here" << std::endl;


@bsunanda<https://github.com/bsunanda> can you please remove commented code?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/7337/files#r23519615>.